src/coreclr/jit/emitwasm.cpp

```diff
@@ -149,22 +156,37 @@ unsigned emitter::instrDesc::idCodeSize() const
     assert(FitsIn<uint8_t>(GetInsOpcode(idIns())));
     switch (idInsFmt())
     {
+        case IF_NONE:
+            return 0;
         case IF_OPCODE:
             break;
         case IF_BLOCK:
             size += 1;
             break;
         case IF_LABEL:
             assert(!idIsCnsReloc());
-            size = SizeOfULEB128(static_cast<target_size_t>(emitGetInsSC(this)));
+            size = SizeOfULEB128(emitGetInsSC(this));
             break;
         case IF_ULEB128:
-            size += idIsCnsReloc() ? PADDED_RELOC_SIZE : SizeOfULEB128(static_cast<target_size_t>(emitGetInsSC(this)));
+            size += idIsCnsReloc() ? PADDED_RELOC_SIZE : SizeOfULEB128(emitGetInsSC(this));
+            break;
+        case IF_SLEB128:
+            size += idIsCnsReloc() ? PADDED_RELOC_SIZE : SizeOfSLEB128(emitGetInsSC(this));
+            break;
+        case IF_F32:
+            size += 4;
+            break;
+        case IF_F64:
+            size += 8;
             break;
         case IF_MEMARG:
-            size += 1; // The alignment hint byte.
-            size += idIsCnsReloc() ? PADDED_RELOC_SIZE : SizeOfULEB128(static_cast<target_size_t>(emitGetInsSC(this)));
+        {
+            uint64_t align = 0; // FIXME
```

Comment:
Let's add a method like `emitGetAlignHintLog2(instrDesc)` with this placeholder value OR we can just implement it right now as `genLog2(EA_SIZE(id->idOpSize()))` and use it here and in `emitDispIns`/`emitOutputInstr`.